docs/usage/configuration-options.md

```diff
@@ -79,6 +79,11 @@ With the above config:
 - ESLint dependencies will have the label `linting`
 - All other dependencies will have the label `dependencies`
 
+<!-- prettier-ignore -->
+!!! note
```

Comment:
```suggestion
!!! warning
```

Maybe a `warning` type admonition is better?

Reply:
I don't think we need to use warning here